ajax/libs/pretty-checkbox/package.json

```diff
@@ -20,9 +20,9 @@
   "npmName": "pretty-checkbox",
   "npmFileMap": [
     {
-      "basePath": "src",
+      "basePath": "dist",
       "files": [
-        "**/pretty*.+(css|js|map)"
+        "**/pretty-checkbox*.+(css|js|map)"
```

Comment:
For dist folder we should use "**/*" pattern

Reply:
Oh okay. Just updated the file.

Reply:
Hi, I am getting this error in CI. "filename from package.json exists and is a file (not a directory)". Can you please help me why this error is thrown?

![image](https://user-images.githubusercontent.com/1754676/32042954-f5edb53a-ba56-11e7-84d8-e01a694e85a5.png)

Reply:
@lokesh-coder Please remove version from package.json or add verson 3.0.0 folder with its assets to you PR